components/helper/ShareButtons.tsx

```diff
@@ -9,9 +12,131 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
```

Comment:
I think that some of these styles are duplicating things in the theme file, so we can pare this down by using those and only adding extra ones if we need to.